src/char/sorceress/blizz_sorc.py

```diff
@@ -199,23 +210,24 @@ def kill_council(self) -> bool:
 
     def kill_nihlatak(self, end_nodes: list[int]) -> bool:
         # Find nilhlatak position
-        atk_sequences = max(2, int(self._char_config["atk_len_nihlatak"]) - 1)
+        atk_sequences = max(1, int(self._char_config["atk_len_nihlatak"]) - 1)
         for i in range(atk_sequences):
             nihlatak_pos_abs = self._pather.find_abs_node_pos(end_nodes[-1], self._screen.grab())
             if nihlatak_pos_abs is not None:
-                cast_pos_abs = np.array([nihlatak_pos_abs[0] * 0.9, nihlatak_pos_abs[1] * 0.9])
-                self._blizzard(cast_pos_abs, spray=90)
-                self._cast_static()
-                # Do some tele "dancing" after each sequence
-                if i < atk_sequences - 1:
-                    rot_deg = random.randint(-10, 10) if i % 2 == 0 else random.randint(170, 190)
-                    tele_pos_abs = unit_vector(rotate_vec(cast_pos_abs, rot_deg)) * 100
-                    pos_m = self._screen.convert_abs_to_monitor(tele_pos_abs)
-                    self.pre_move()
-                    self.move(pos_m)
+                cast_pos_abs = np.array([nihlatak_pos_abs[0] * 1.0, nihlatak_pos_abs[1] * 1.0])
+                wait(0.8)  
+                self._blizzard(cast_pos_abs, spray=0)
+                wait(0.3)  
+                nihl_immune = self._template_finder.search(["COLD_IMMUNE","COLD_IMMUNES"], self._screen.grab(), threshold=0.8, roi=self._config.ui_roi["enemy_info"]).valid
+                if Nihl_immune:
```

Comment:
you also have to adapt this variable, otherwise it will throw an error on execution.